neqo-crypto/src/ext.rs

```diff
@@ -99,7 +102,8 @@ impl ExtensionTracker {
         arg: *mut c_void,
     ) -> SECStatus {
         let d = std::slice::from_raw_parts(data, len as usize);
-        ExtensionTracker::wrap_handler_call(arg, |handler| {
+        Self::wrap_handler_call(arg, |handler| {
+            // Cast is safe here because the message type is always part of the enum
             match handler.handle(message as HandshakeMessage, d) {
```

Comment:
Hmm, this is one of those cases where the C bindings produce a u32 where a u8 would suffice.  I wonder if we can't fix the bindings somehow.

Reply:
I'll read some bindgen and co documentation and try to figure out, that would be much cleaner indeed !

Reply:
opened #168